README.md

```diff
@@ -48,8 +43,8 @@ Or, if you use the static release, try following
 </script>
 ```
 
-The argument `clientProperties` object is mandatory for the constructor.
-It is an example in which a user `foo@example.com` tries to connect to the server `scalardl.example.com:50051` of the Scalar DL network.
+The `clientProperties` argument is mandatory for the constructor.
+This is a properties example that a user `foo@example.com` would use to try to connect to the server `scalardl.example.com:50051` of the Scalar DL network.
```

Comment:
I wonder if we can point people to use the Sandbox environment here.  @feeblefakie what do you think?

```suggestion
Take a look at the example `clientProperties` below to see how to connect to the Scalar DL network. In the example the user `foo@example.com` is connecting to the server `scalardl.example.com` on port `50051` with a private-key and certificate.
```